src/com/hms_networks/americas/sc/mqtt/MqttManager.java

```diff
@@ -25,6 +26,15 @@
   /** Boolean flag to control the MQTT thread loop. */
   private boolean mqttThreadRun = true;
 
+  /**
+   * Boolean flag to enable UTF8 string conversion. Will enable non-ascii characters, but decreases
+   * performance by a significant amount. Defaults to false, and can be set by constructor.
+   */
+  private boolean mqttUtf8Convert = false;
+
+  /** Encoding format for strings */
+  private final String UTF_8 = "UTF-8";
```

Comment:
```suggestion
  private static final String UTF_8 = "UTF-8";
```
This variable should be a 'static final' constant. The ALL_CAPS naming convention only applies to 'static final' constants as well.